### ISSUE TITLE
Improve Seaside control panel initialisation

### ISSUE THREAD

**Issue by theseion** [SeasideSt/Seaside#1074]
Neither in Squeak nor Pharo a default adapter is installed. In Squeak, loading with the Github invocation doesn't even install any usable adaptors. The following is what the SqueakMap entry does:

```Smalltalk
SMSqueakMap default loadUpdates.

Installer ensureRecentMetacello.

(Smalltalk at: #Metacello) new
 baseline:'Seaside3';
 repository: 'github://SeasideSt/Seaside:master/repository';
 load.

(Installer ss3 project: 'WebClient')
        install: 'WebClient-Seaside-Adaptor'.

"Optionally use the control panel to add adaptor, start and to set encoding"
"WAControlPanel open."

"do the above but without using the GUI"
((Smalltalk at: #WAWebServerAdaptor) port: 8080)
        codec: ((Smalltalk at: #GRCodec) forEncoding: 'utf-8');
        start.
```

**Comment by jbrichau** [comment]
Related: Pharo in https://github.com/SeasideSt/Seaside/pull/1096

**Comment by jbrichau** [comment]
I added the initialisation to the BaselineOf for Squeak as well. Fixed as part of 3.3.0
